ooniprobe/Model/Database/Result.mm

```diff
@@ -18,14 +18,37 @@ - (SRKResultSet*)measurements {
 }
 
 - (SRKResultSet*)measurementsSorted {
-    //Not showing the re_run measurements
-    return [[[[[[Measurement query]
-               where:@"result_id = ? AND is_rerun = 0 AND is_done = 1"
-               parameters:@[self]]
-               orderByDescending:@"is_anomaly"]
-               orderByDescending:@"is_failed"]
-               order:@"Id"]
-               fetch];
+    /*
+     * Sorting measurements:
+     * by is_anomaly and is_failed for Websites
+     * Whatsapp - Telegram - Facebook for Instant Messaging
+     * Ndt - Dash - HIRL - HHFM for Performance
+     * Psiphon - Tor for Circumvention
+    */
+    if ([self.test_group_name isEqualToString:@"websites"]){
+        return [[[[[[Measurement query]
+                   where:@"result_id = ? AND is_rerun = 0 AND is_done = 1"
+                   parameters:@[self]]
+                   orderByDescending:@"is_anomaly"]
+                   orderByDescending:@"is_failed"]
+                   order:@"Id"]
+                   fetch];
+    }
+    NSMutableArray *measurementsSorted = [NSMutableArray new];
+    SRKResultSet *measurements = self.measurements;
+    NSArray *testOrder = [[TestUtility getTests] objectForKey:self.test_group_name];
```

Comment:
where's the testOrder? is it already in master?

Reply:
yes, it's in the `TestUtility` class